dmaws/utils.py

```diff
@@ -3,7 +3,6 @@
 
 import yaml
 import jinja2
-from jinja2.exceptions import TemplateSyntaxError, UndefinedError  # noqa
```

Comment:
Hmm, we seem to have left these in for a reason - another file imports them

Reply:
I couldn't find where else imports TemplateSyntaxError. As for UndefinedError, I've just added that specific import into test_utils rather than importing it to here and then importing it from here to there